packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -466,17 +466,71 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         // snapshot code path. We should leverage the fact that these deltas are returned to speed up the deltas fetch.
         const { headers, url } = getUrlAndHeadersWithAuth(`${this.snapshotUrl}/trees/latest${snapshotOptions}`, storageToken);
 
+        let isOptionsCall = false;
+
+        if (Object.keys(headers).length) {
```

Comment:
The browser fetch implementation typically hides options from you. Are we sure this works?

Reply:
These headers are the request header which we constructed. So they should not be hidden.

Reply:
Yes, this works - I've verified this on localhost:8081/ by running npm run start:spo-df in examples/data-objects/shared-text and using F12 to step thru source code and checking for length value.

Reply:
Oh I misunderstood the intent here based on the name. This won't actually tell you if the browser made a blocking options call, since those can be  cached and omitted. The name isOptionsCall made me think this was measuring if we actually made an options network request, but in practice, it is saying we made a request that may require an options network request, if it isn't cached.

Reply:
Ideally, we want to measure if we actually made an options network request but were unable to figure out how, so went with current approach. If there is a way to figure out if we actually made an options call, pl. let us know.